shared/1ds-core-js/test/Unit/src/FileSizeCheckTest.ts

```diff
@@ -51,8 +51,8 @@ function _checkSize(checkType: string, maxSize: number, size: number, isNightly:
 }    
 
 export class FileSizeCheckTest extends AITestClass {
-    private readonly MAX_BUNDLE_SIZE = 70;
-    private readonly MAX_DEFLATE_SIZE = 30;
+    private readonly MAX_BUNDLE_SIZE = 80;
```

Comment:
Yes, ouch a 10kb jump -- but this is the unminified file which also includes all of the comments